system/jlib/jlog.hpp

```diff
@@ -836,6 +836,7 @@ constexpr LogMsgCategory MCdebugInfo(MSGAUD_programmer, MSGCLS_information, Debu
 constexpr LogMsgCategory MCauditInfo(MSGAUD_audit, MSGCLS_information, AudMsgThreshold);
 constexpr LogMsgCategory MCoperatorInfo(MSGAUD_operator, MSGCLS_information, InfoMsgThreshold);
 constexpr LogMsgCategory MCoperatorMetric(MSGAUD_operator, MSGCLS_metric, ErrMsgThreshold);
+constexpr LogMsgCategory MCoperatorTrace(MSGAUD_monitor, MSGCLS_event, ProgressMsgThreshold);
```

Comment:
This is needed regardless, could be added in a dedicated PR if we feel it's appropriate to do so.

Reply:
addressed here: https://track.hpccsystems.com/browse/HPCC-30696